python/semantic_kernel/connectors/memory/astradb/exceptions.py

```diff
@@ -0,0 +1,40 @@
+class AstraDbException(Exception):
```

Comment:
Can this be based on the existing exception classes within SK?

Reply:
done